tests/kuttl/tests/dataplane-create-test/00-assert.yaml

```diff
@@ -36,197 +12,86 @@ metadata:
     kind: OpenStackDataPlane
     name: openstack-edpm
 spec:
-  dataPlane: openstack-edpm
```

Comment:
this would need to be openstack-edpm-ipam since you symlinked to a different file.

Reply:
Updated